src/markdown-pages/build-apps/create-an-ab-test-application/publish.mdx

```diff
@@ -209,8 +205,8 @@ Submit your catalog information:
 
 ```sh
 nr1 catalog:submit
-[output] Uploading screenshots...
-[output]  {green}✔  {plain}Screenshots uploaded
+[output] Uploading screenshots from nru-programmability-course/publish/ab-test...
+[output]  {success}✔{normal}  Screenshots uploaded from: nru-programmability-course/publish/ab-test
```

Comment:
Pretty sure this section should also include the output from the `launcher` + `nerdlet` in this repo, yeah?

Similar to what we've got over here: https://github.com/newrelic/developer-website/pull/1597/files#diff-a5b87ebe2e68e66c60aff11ad881315a991250324cd72f76a96165187dc8ab49R181-R188

Course repo for reference: https://github.com/newrelic-experimental/nru-programmability-course/tree/main/describe-app/ab-test

Reply:
Let me look at this again. I was under the impression that if the screenshots folder was empty, we wouldn't try to upload it, so let me make sure that is still the case, hence why I omitted that output here. The course doesn't call for adding screenshots at the nerdpack item level, so I only included output from the root-level screenshots.

Reply:
Ok after trying this out some more on my own, it looks like we actually don't skip screenshots on empty folders, which means that it errors out when trying to submit this step.

```sh
$ nr1 catalog:submit
Uploading screenshots from /path/to/demo-app...
 ✔  Screenshots uploaded from: /path/to/demo-app
Uploading screenshots from /path/to/demo-app/nerdlets/home...
Uploading screenshots from /path/to/demo-app/launchers/launcher...
 ›   Error: 1 error while updating Home 1.0.0
 ›
 ›   "You must include at least 1 screenshot when uploading screenshots."
 ›   Code: UNKNOWN
```

Reply:
Ah, I see. Good point. All good to leave as is if that's how the command behaves. 

Pretty sure we're skipping the upload on the root directory, but we never established that for the Nerdpack-item children `screenshots/` directories, which feels like an oversight

Reply:
So are these Nerdpack item-level screenshots mandatory or can we delete `screenshots/` if there aren't any?

Reply:
@alexronquillo per [this comment](https://github.com/newrelic/developer-website/pull/1597#discussion_r706387907), they are not required and we are releasing a patch to skip uploading screenshots directories that are empty. The output that I have above will be the correct one once that patch goes live.